packages/lodestar/src/sync/unknownBlock.ts

```diff
@@ -164,7 +164,9 @@ export class UnknownBlockSync {
     }
 
     pendingBlock.status = PendingBlockStatus.processing;
-    const res = await wrapError(this.chain.processBlock(pendingBlock.signedBlock, {ignoreIfKnown: true}));
+    const res = await wrapError(
+      this.chain.processBlock(pendingBlock.signedBlock, {ignoreIfKnown: true, blsVerifyMainThread: true})
```

Comment:
Add explanation above the blsVerifyMainThread option linking to the issue and investigation that motivated this change. Copy paste from `packages/lodestar/src/network/gossip/handlers/index.ts`.

Then please add a note why unknown block sync should use true and range sync false